ministryofjustice/analytics-platform-control-panel-frontend
# Give app access to a s3bucket through the UI

## What

Connect S3 buckets to apps through the UI

## How to review

1. check out, run and log in
2. have at least one app and at least one S3 bucket in your local DB
3. visit `/apps` and select your app
4. see the dropdown for connecting buckets to the app

![screen shot 2017-11-08 at 12 45 34](https://user-images.githubusercontent.com/988436/32549829-c497d16a-c482-11e7-8eb4-f04d78011411.png)

5. pick one and click "Add"
6. experience joy

![screen shot 2017-11-08 at 12 46 19](https://user-images.githubusercontent.com/988436/32549851-d8d3cf62-c482-11e7-9805-f8aa74e01a28.png)
